x-pack/plugin/esql/qa/testFixtures/src/main/resources/ip.csv-spec

```diff
@@ -146,7 +146,7 @@ fe80::cae2:65ff:fece:fec1                             |[fe80::cae2:65ff:fece:feb
 ;
 
 in
-required_capability: mv_warn
+required_capability: in_evaluatormapper
```

Comment:
My understanding is that, unlike features and transport version, having a capability does not imply having all older capabilities.  So, maybe we want this to require both `mv_warn` and `in_evaluatormapper`?  Although I am just taking it on faith that `required_capability` can accept more than one requirement.

Reply:
You write `required_capability` twice if you want two.

Reply:
If the test returns different results between the two `required_capability`, what should I here?

Reply:
> If the test returns different results between the two required_capability, what should I here?

We're not removing the old capability so you could use both. Or just the one added after. Either would work and I'm not sure I have an opinion yet on which of those two choices is better.

Reply:
> > If the test returns different results between the two required_capability, what should I here?
> 
> We're not removing the old capability so you could use both. Or just the one added after. Either would work and I'm not sure I have an opinion yet on which of those two choices is better.

I'll replace the old one with the new one for now, as the results changed...